.github/workflows/test-expose-port.yml

```diff
@@ -0,0 +1,56 @@
+name: test - expose-port
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - expose-port/**
+      - .github/workflows/test-expose-port.yml
+
+concurrency:
+  group: test-expose-port-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: "npm"
+          cache-dependency-path: |
+            **/package-lock.json
+
+      - name: build
+        run: make package-expose-port
+
+      - name: create-cluster
+        id: create-cluster
+        uses: replicatedhq/compatibility-actions/create-cluster@v1
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          kubernetes-distribution: kind
+          timeout-minutes: 5
+
+      - name: test expose-port
```

Comment:
Idea: before exposing a port, run a `kubectl apply` and install nginx with a nodeport service.
After `expose-port`, verify with `curl` the exposed port.
Or would this be beyond testing the action?

Reply:
We test this in cmx e2e tests